README.md

```diff
@@ -21,20 +21,27 @@ local directory.
 $ dhttpd
 ```
 
-Otherwise you can use the `pub global` command.
+Otherwise you can use the `dart pub global` command.
 
 ```console
-$ pub global run dhttpd
+$ dart pub global run dhttpd
 ```
 
-Here's an example of creating a web app with [Stagehand](https://pub.dev/packages/stagehand)
+Activate [webdev](https://dart.dev/tools/webdev) to enable dart web application builds.
+
+```console
+$ dart pub global activate webdev
+```
+
+Here's an example of creating a web app
 and then running it with this server:
 
 ```console
-$ stagehand web-angular
-$ pub get
-$ pub build
-$ dhttpd --path build/web/  # Serves app at http://localhost:8080
+$ dart create -t web-simple web-app
+$ cd web-app
+$ dart pub get
+$ webdev build
```

Comment:
This should be `dart build` or `dart run build_runner build` or something.

Did you test this?

Reply:
Yes, this was tested. This is what some of the documentation recommended, but I see now that this is for a development flow. Updated with your recommended build_runner.